field.go

```diff
@@ -1203,9 +1243,9 @@ func (f *Field) importValue(columnIDs []uint64, values []int64, options *ImportO
 			return errors.Wrap(err, "creating fragment")
 		}
 
-		baseValues := make([]uint64, len(data.Values))
+		baseValues := make([]int64, len(data.Values))
 		for i, value := range data.Values {
-			baseValues[i] = uint64(value - bsig.Min)
+			baseValues[i] = int64(value - bsig.Min)
```

Comment:
we're kind of repurposing `bsiGroup.Min` now, right? Like previously it actually meant the minimum value stored in the field, but not it means the "base" value which the which the positive or negative stored values are  added to?

If that's correct, it might be worth renaming (not sure if it's exposed anywhere) or at least adding a comment on the struct field and in key places of use.